src/Modules/OrchardCore.Commerce/Startup.cs

```diff
@@ -99,6 +99,9 @@ public override void ConfigureServices(IServiceCollection services)
         services.AddScoped<IShoppingCartHelpers, ShoppingCartHelpers>();
         services.AddScoped<IShoppingCartSerializer, ShoppingCartSerializer>();
         services.AddActivity<ProductAddedToCartEvent, ProductAddedToCartEventDisplay>();
+        services.AddScoped<IDataMigration, ShoppingCartWidgetMigrations>();
+        services.AddContentPart<ShoppingCartWidgetPart>()
+            .UseDisplayDriver<ShoppingCartWidgetPartDisplayDriver>();
```

Comment:
It's neater this way when the migration is specific to the part:

```suggestion
        services.AddContentPart<ShoppingCartWidgetPart>()
            .UseDisplayDriver<ShoppingCartWidgetPartDisplayDriver>()
            .WithMigration<ShoppingCartWidgetMigrations>();
```